cmd/bosun/sched/sched.go

```diff
@@ -43,6 +43,7 @@ type Schedule struct {
 	LastCheck     time.Time
 	nc            chan interface{}
 	notifications map[*conf.Notification][]*State
+	lastLogTimes  map[expr.AlertKey]time.Time
```

Comment:
I thought about persisting it and decided it probably isn't worth saving. Perhaps @gbrayut would have input there. Locking is a good point. 

Reasoning for a schedule map is that it will likely consume less memory since it will usually be pretty sparse. I could go either way though.